shared-library.cmake

```diff
@@ -16,10 +16,11 @@
 # Shared library related constants
 # (used for pkg-config file generation).
 # FIXME: can't we get these information from CMake directly?
+SET(LIBDIR_KW "-L")
+SET(LIBINCL_KW "-l")
+SET(LIBINCL_ABSKW "")
+SET(LIB_EXT "")
```

Comment:
@jcarpent Those default values are fine for Mac ?

Reply:
I will try them indeed.

Reply:
Seems fine.